doc/sphinx/constraints.rst

```diff
@@ -450,6 +451,36 @@ Note: in the OpenGL visualizer, if the OpenGL Extrusion library is not available
 the shape surface will be rendered with dots.
 
 
+Quarterpipe
+""""""""""""
+
+:class:`espressomd.shapes.Quarterpipe`
+
+A quarterpipe which can be used to connect two walls or rhomboids in a smooth way.
+The shape is constructed by removing a quarter cylinder of radius ``radius``
+from a cuboid with side length ``radius``.
+It can be created via ::
+
+    shape=espressomd.shapes.Quarterpipe(center=[25]*3, axis=[1, 0, 0],
+                                        orientation=[0, 1, 1], radius=25,
+                                        height=30)
+                                         
+A cross section with an explanation of the parameters is shown in the figure below.
+The three-dimensional shape is an extrusion of this profile.
+
+.. figure:: figures/quarterpipe.png
+   :alt: Cross section of the quarterpipe shape.
+   :height: 4.00000cm
+
+``center`` defines the  center of the cylinder. 
+``axis`` is the axis of the cylinder,
+orthogonal to the curve of the pipe.
```

Comment:
this linebreak looks odd to me because the line below is even longer
```suggestion
``axis`` is the axis of the cylinder, orthogonal to the curve of the pipe.
```